scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Record.java

```diff
@@ -521,12 +525,13 @@ private Record(ProtocolVersion version, int epoch, long sequenceNumber) {
 		plaintext.writeBytes(getBlockCipherMac(writeState, compressedFragment));
 
 		// determine padding length
-		int ciphertextLength = compressedFragment.length + writeState.getCipherSuite().getMacLength() + 1;
```

Comment:
writeState.getCipherSuite().getMacLength() and writeState.getMacLength() should contains the same value right ? 

(it's just to make the code clearer?)

Reply:
yes.